roles/jboss_eap/tasks/main.yml

```diff
@@ -23,3 +23,8 @@
       register: jboss.eap.processes
       ignore_errors: yes
       when: '"jboss.eap.processes" in facts_to_collect'
+    - name: check for jboss packages
+      raw: rpm -qa --qf "%{NAME}|%{VERSION}|%{RELEASE}|%{INSTALLTIME}|%{VENDOR}|%{BUILDTIME}|%{BUILDHOST}|%{SOURCERPM}|%{LICENSE}|%{PACKAGER}|%{INSTALLTIME:date}|%{BUILDTIME:date}\n" | grep -E '(eap7)|(jbossas)' | sort
```

Comment:
Based on your middleware EAP fingerprinting section are we capturing all versions here?
(eap6)?

Reply:
EAP6 used "jbossas" as its package prefix, oddly.

Reply:
And to be clear, jbossas is there only to capture EAP 6. The EAP 7 packages all have "eap7" in their name.